app/src/androidTest/java/com/example/bookwormadventuresdeluxe2/MyBooksFragmentTest.java

```diff
@@ -43,7 +43,7 @@
     private Resources resources;
 
     private int BACK_BUTTON_INDEX = 0;
-    /* In the edit book fragment the index is different because android is stupid and picks
+    /* In the edit book fragment the index is different as android picks
```

Comment:
Lol nice catch

Reply:
this was my comment I was getting mad when writing that test lol

Reply:
Haha 😂